api/app/events/models.py

```diff
@@ -0,0 +1,55 @@
+
+from app import db
+
+class Event(db.Model):
+
+    __tablename__ = "Event"
+
+    id = db.Column(db.Integer(), primary_key=True)
+    name = db.Column(db.String(50))
+    description = db.Column(db.String(255))
+    startDate = db.Column(db.DateTime())
+    endDate = db.Column(DateTime())
```

Comment:
Something else I noticed is that the non-primary key columns are set to be nullable by default. Reading through the spec again now, it seems that these columns should always be required. Can we set this in the database?

You can do so by setting a nullable parameter as follows
`name = db.Column(db.String(50), nullable=True)`

You will need to re-run the migration though to get the updates.

Reply:
Applied this change and reran the migrations